### DISCUSSION TITLE
The White Skull revamp

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#68] [Card Balance]
I have been thinking a lot about this so this is not rushed suggestion. Bauhaus has nice working strategy with Etoiles Mortants, but Rangers are sort of overlooked. 

Venusian Ranger
3/4/4/4
VENUSIANRANGER
+2/+2/+2 when attacking

I think their current skill reflects well their style of combat. It represents surprise ambush attack. Only change i have made is swapped Fight and Shoot value, in contrast with close combat oriented Etoiles Mortants.

`The most famed Bauhaus regiment in the Solar System, the Venusian Rangers are excellent marksmen and silent hunters.
Equipment: AG-17 assault rifle and MP-105 machine pistol.`

Venusian Marshall
4/7/7/7
VENUSIANRANGER BOUNTY
FIRSTSTRIKE when equiped with FIREARM

Venusian Marshall is excellent marksman in tabletop. This design reflects his markmanship. He becomes centerpiece of your squad and not support warrior. With AG-17 double attack he can take down very strong enemy and survive second attack thanks to First strike. Or he can finish up wounded warriors and escape without harm, while collecting extra VP with Bounty.

`Venusian Marshalls are sent out on missions too harsh or too difficult to take on by foot or vehicle. They ride sturdiest of mounts - bred to endure those harsh conditions. If it is your dream to wear legendary white skull facemask of the rangers - service as Marshall is sure to wake you up. The Venusian Marshall rides a horse. There is one statline for both horse and the rider to show how hard is to discern the two on the battlefield. 
Special rule: Terrain Jumping. 
Equipment: Kampfkanone and Violator sword.`

Max Steiner
4/7/9/8
UNIQUE VENUSIANRANGER
All your non-unique VENUSIANRANGERS gain +2/+2/+2.
1 Action: Target VENUSIANRANGER gains an AMBUSH until end of your turn.

(Alternatively instead of +2/+2/+2 bonus: Max Steiner ignores any attack restrictions and can attack any warrior from opponents squad) 

AMBUSH: If your warrior wounds an opponent, he doesnt retaliate.

Max Steiner gets Marshall's passive ability +2/+2/+2 to boost other rangers, but since everyone says +X abilities are boring there is alternative for more tabletop inspired ability. He can ignore frontline and other restrictions, which would otherwise prevent him to attack some particular warrior.
Similarly to Sean Galagher gives Slayer to fellow Clansmen, his active ability will provide "Surprise Attack" effect to target Rangers.

`Max is wild man plain and simple. He is almost completely free of inhibitions, and he loves it. For Max attitude is everything. He cares deeply about portraying himself in just the right light at all times. He is entirely egocentric, believing he is coolest guy around. The worst part is, he's usually right. Despite his cultivated veneer of disrespect and irresponsibility, Max is formidable man and one of Bauhaus' finest. 
The blonde hair, the blue eyed fast talker with and immature glint in his eyes has kind of boyish charm about him that annoys as many people as it impresses. Those not fascinated by him usually find him obnoxious. Max carries his venusian ranger's battlemask as often as possible, as he feels it adds to his tough-guy image. (which since the rangers are one of the Bauhaus' most elite special forces unit - it does.)
Special rule: Because Max has unpredicable side to him, he can shoot any model within range. (Not the closest model)
Equipment: Two Plasma Handcannons, one in each hand and Violator sword for close combat.`

**Comment by framerate** [comment]
I really like "first strike when equipped with a firearm". Thats unique and encourages new ways to play. Some good stuff in here. I'll give it another read but tend to agree with a lot of this!

**Reply by Sol-au-fein** [reply]
Awesome! In case of Marshall, i was worried if that would slide, because It is significant change. People generally complained he was too strong.
